code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -178,6 +178,21 @@
 		M.update_inv_l_hand()
 		M.update_inv_r_hand()
 
+/obj/item/reagent_containers/syringe/Crossed(AM as mob|obj, oldloc)
+	var/mob/living/carbon/human/H = AM
```

Comment:
Why not scope this just to /mob/living/carbon/human/AM in the proc to cut the as + reasignment and to simplify this to just an istype() check

Reply:
its best to just be consistent with the parent proc

Reply:
because i'm on my evil grind

Reply:
the ishuman later in this proc can just be changed to istype in that case, as it already has scoped itself to human